### PR TITLE
Change default value of 'closed' to match Glyphs 3 behaviour

### PR DESCRIPTION
**Pull request by Hoolean**
Hi Georg!

We found that omitting the `closed` property causes Glyphs 3 to assume a path is closed; can you confirm that this is what Glyphs expects in all scenarios?

(if the Glyphs 2 file format behaves the same way, then am happy to include that change in this mini-PR too 🤜🤛)

### CODE REVIEW

**Comment by schriftgestalt** [comment]
Glyphs will always write `closed`. But if it is missing, both Glyphs 2 and 3 will assume "closed = 1" (meaning closed).

But please send the PR to the Glyphs3-merge branch.

**Comment by Hoolean** [comment]
Thanks for confirming and the speedy reply :)

I have added a commit that makes the same change for the Glyphs 2 docs. At the moment this change is for internal DaMa tooling and independent of glyphsLib, and so I think it is targeting the right branch for a docs-only change on this repo, but let me know if you want me to point it [at `clean-up` or `master` instead](https://github.com/schriftgestalt/GlyphsSDK/branches/all).

The PR should now be ready for merge, unless we would prefer to rephrase it to clarify that Glyphs.app (and maybe other tools?) should always write the value.

**Comment by schriftgestalt** [comment]
I fixed the wording of the line.
